server/src/helpers/responses.js

```diff
@@ -0,0 +1,17 @@
+const responses = {};
+
+responses.errorProcessing = (req, res) => {
+  return res.status(500).json({
```

Comment:
The status code doesn't really match the error message. 5xx are basically for server error.

Reply:
Resolved